.eslintrc.json

```diff
@@ -16,7 +16,8 @@
     "@typescript-eslint/no-extra-semi": "off",
     "@typescript-eslint/no-empty-function": "off",
     "simple-import-sort/imports": "error",
-    "simple-import-sort/exports": "error"
+    "simple-import-sort/exports": "error",
+    "@typescript-eslint/no-namespace": ["warn", { "allowDeclarations": true }]
```

Comment:
warn -> error

why is this check needed?

Reply:
It's not needed IMHO. It's `error` by default without exemption for declarations, by I wanted to use `export declare namespace` to attach types to a class with declaration merging.